### ISSUE TITLE
Question about your steps

### ISSUE THREAD

**Issue by Pixelherz23** [twisted89/Manor-Lords-Save-Load-Mod#1]
What do you mean with: "Copy release folder into root game install folder (Manor Lords Demo) next to ManorLords.exe"
Whats the release  folder and whats the root game install folder?

**Comment by twisted89** [comment]
Where you installed the game...

**Comment by Pixelherz23** [comment]
Then I don't understand the step. Could you rephrase the sentence for me, please?
What should I copy into the "Manor Lords Demo" folder?
Thanks!

**Comment by twisted89** [comment]
I'm not sure how to make it any clearer, find the location of ManorLords.exe and copy it into the same folder.

**Comment by Pixelherz23** [comment]
So I am in the folder where ManorLords.exe is located. Now what should I copy into this folder?

**Comment by Pixelherz23** [comment]
"Step 3: **Copy release folder into** root game install folder (Manor Lords Demo) next to ManorLords.exe"

What is the share folder if the "root game install folder" is the folder where ManorLords.exe is located?
That what confuses me

**Comment by twisted89** [comment]
Releases: https://github.com/twisted89/Manor-Lords-Save-Load-Mod/releases

**Comment by Pixelherz23** [comment]
I appreciate your work, thanks man! Saving and loading works now.
